spec/stat_tracker_spec.rb

```diff
@@ -24,9 +24,13 @@
       p @stat_tracker
     end
 
+    it 'processed team data, retrieves data from teams' do
```

Comment:
We will need to update this test once with mock data.

Reply:
- [x]